sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -182,13 +191,26 @@ export default defineComponent({
         StudentService.shared.getStudentProfile(),
         ApplicationService.shared.getApplicationData(props.id),
       ]);
-      // Adjust the spaces when optional fields are not present.
-      isReadOnly.value =
-        [
+
+      // Define if the form should be read-only.
+      if (props.readOnly) {
+        // If some value is provided in the read-only prop, then the form should be readonly.
+        isReadOnly.value = true;
+      } else if (
+        // If a change request is being made in the expected application status,
+        // then the form should be editable.
+        props.changeRequest &&
+        applicationData.applicationStatus === ApplicationStatus.Completed
```

Comment:
As a UI we do not route to change request when the application status is not completed, so is the second condition on applicationStatus required here?

Reply:
It is not to prevent internal Vue routes but from the ability of the user to change a parameter directly in the UI.